### PR TITLE
fix(package): Set typescript version as ~2.3.4 instead of ^2.2.1 and …

### PR DESCRIPTION
**Pull request by cojack**
…ts-node up to ~3.2.0 from ^2.0.0

### CODE REVIEW

**Comment by cojack** [comment]
It's related to issue #102

File: package.json [hunk 1]
```diff
@@ -56,7 +56,7 @@
     "reflect-metadata": "^0.1.10",
     "rxjs": "^5.4.0",
     "socket.io": "^1.7.2",
-    "typescript": "^2.2.1"
+    "typescript": "~2.3.4"
```
Comment (d3viant0ne) [inline package.json]: Out of curiosity, why lock down the TypeScript dependency to patch? They have for the most part been much better about not breaking things with `semver: Minor` ( excluding the `rxjs / ts@2.4.x` thing )

Comment (cojack) [inline package.json]: @d3viant0ne because 2.4 have breaking changes that prevents Nest from work, but I don't think so Kamil with merge this PR.

**Comment by kamilmysliwiec** [comment]
It's changed in v4 :smiley: thanks @cojack!

**Pull request closed without merging**